hail/python/hailtop/utils/utils.py

```diff
@@ -116,7 +116,7 @@ def grouped(n: int, ls: List[Any]) -> Iterable[Any]:
         yield group
 
 
-def partition(k, ls):
+def partition(k: int, ls: List[int]) -> Iterable[Any]:
```

Comment:
What is the type of the object we are yielding on line 134? Replace `Iterable[Any]` with `Iterable[.......]` where `......` is the type of the object on line 134 being yielded.